cookbook/RAG_multi_modal.ipynb

```diff
@@ -0,0 +1,1051 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "0eee25ed-f37c-4d53-abc7-b9f2c3ef4882",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from lxml import html\n",
+    "from pydantic import BaseModel\n",
+    "from typing import Any, Optional\n",
+    "from unstructured.partition.pdf import partition_pdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74b56bde-1ba0-4525-a11d-cab02c5659e4",
+   "metadata": {},
+   "source": [
+    "## Data Loading\n",
```

Comment:
Can a use case description be added? What is the problem, and how is it solved here? 
TNX

Reply:
ya, this is just a draft. will be adding that, and this will publish w/ a blog post.